hack/update-openapi-spec.sh

```diff
@@ -33,8 +33,8 @@ make -C "${KUBE_ROOT}" WHAT=cmd/kube-apiserver
 function cleanup()
 {
     if [[ -n ${APISERVER_PID-} ]]; then
-      kill ${APISERVER_PID} 1>&2 2>/dev/null
-      wait ${APISERVER_PID} || true
+      kill "${APISERVER_PID}" 1>&2 2>/dev/null
+      wait "${APISERVER_PID}" || true
```

Comment:
change reason:
```
In hack/update-openapi-spec.sh line 36:
      kill ${APISERVER_PID} 1>&2 2>/dev/null
           ^-- SC2086: Double quote to prevent globbing and word splitting.


In hack/update-openapi-spec.sh line 37:
      wait ${APISERVER_PID} || true
           ^-- SC2086: Double quote to prevent globbing and word splitting.
```